fabfile.py

```diff
@@ -1528,18 +1519,16 @@ def get_vendor_config():
     return config
 
 
-@task
 def update_vendor_themes(frontend_version=1):
-    """Update optional themes in assembl/static/css/themes/vendor"""
     sanitize_env()
-    config = get_vendor_config()
-    config_section_name = 'theme2_repositories' if frontend_version == 2 else 'theme_repositories'
-    if config.has_section(config_section_name):
+    assert frontend_version in (1, 2)
+    theme_varname = "theme%d_repositories__git-urls" % frontend_version
```

Comment:
It's not in a config file, it's an argument to the function. I kept the fabfile assumption that both always have to be updated.